aws/aws.go

```diff
@@ -29,11 +29,12 @@ const topicNamePrefix = "pubsub__"
 // AWS SQS queue names cannot exceed 80 characters. The naming convention for
 // queues is to have the prefix+topic+subscriptionID, so we have to share these
 // 80 characters between the 3 items.
-const subscriptionIDMaxLength = 40
+const subscriptionIDMaxLength = 20
 
 // AWS SNS topic names can be up to 256 characters, but because of our naming
 // convention we're limited to the total 80-character max of the SQS name length.
-const topicNameMaxLength = 40 - len(topicNamePrefix)
+// For topic name we have 51 character: 80-8(prefix)-1(dash)-20(subscriptionID)
+const topicNameMaxLength = 51 - len(topicNamePrefix)
```

Comment:
Isn't `51 - len(topicNamePrefix)` double counting the prefix length?

Reply:
@Calebjh no because the structure is as follows: 
prefix+topic+subscriptionID
**pubsub__**(8) + **topic_name**(51) + **_**(1) + **subsID**(20)

Reply:
@Calebjh is right need to change that line because you are double counting the prefix length.

```suggestion
const topicNameMaxLength = `80 - len(topicNamePrefix) - 1 - subscriptionIDMaxLength`
```